tools/internal_ci/linux/grpc_xds_k8s_run_xtest.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## xDS test server/client Docker images
+readonly IMAGE_REPO="gcr.io/grpc-testing/xds-interop"
+
+find_latest() {
+  gcloud container images list-tags --filter='tags~v1\.\d+\.x' --flatten='tags[]' --format='value(tags)' | sort --version-sort | tail -n 1
+}
+
+if [ "${LATEST_BRANCH}" == "" ]; then
```

Comment:
I highly recommend to create a function. Then it's much clearer what's happening. Side-effects should be avoided as much as possible.

> Put all functions together in the file just below constants. **Don’t hide executable code between functions.** Doing so makes the code difficult to follow and results in nasty surprises when debugging.  
> If you’ve got functions, put them all together near the top of the file. **Only includes, set statements and setting constants may be done before declaring functions.**  
> — https://google.github.io/styleguide/shellguide.html#function-location

Reply:
done